src/index.html

```diff
@@ -6,8 +6,58 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__rectangle">
+
+        <img
+          src="images/imac.jpeg"
+          class="card__img"
+          alt="imac"
+        >
+
+        <header class="card__description">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </header>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__price-bloc">
+          <div class="card__price-word">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <div class="card__button">
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="hover"
+          >
+          Buy
+          </a>
+        </div>
+
+        <div class="rate">
+          <div class="stars stars--2">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="rate__reviews">Reviews: 5</p>
+        </div>
```

Comment:
Structurally, it should be higher and it should not be stylized due to absolute positioning

Reply:
if I understood correctly, I fixed the code to meet these requirements